cityofaustin/joplin
# 4016 Joplin UI: Application Banner & Primary Navigation

Zenhub issue: https://app.zenhub.com/workspaces/techstack-5a78b88e1ce69f3510b678ef/issues/cityofaustin/techstack/4016

Design specs: https://xd.adobe.com/view/c533a077-9314-46a6-5db4-d4a7f8601e8f-b302/

Documentation: https://docs.google.com/document/d/1SdAro51Ppdi3Hirn1XkVviJ6nu89dDXtvvFKIX4T8t8/edit#heading=h.ff2tre8z4na1

# Description


# Summary 
 This PR is the first actual step toward building out our new Joplin UI. In the PR I attempted to do three things. 
1. Create a toggle-able site for two different UIs (Default: old).
  -  Allows our current UI to persist without change, 
  -  And, the new UI can be built without tangling with the old css/html/js. 
2. Organize and create a model for how we can develop the remainder of the new UI AND a best practice going forward.
3. Complete a first pass at implementing the Application Banner and Primary Navigation.

# Types of changes

# How can this be tested?

Deployed: https://joplin-pr-4016-navs.herokuapp.com/
- click the lamp in the center of the city seal. It's small, so you may have to  click around  a bit (10x10px)
- This PR only covers TWO sections of the new design. 
  - The Primary Navigation ("Side Nav")
  - The Application Banner, including User Account dropdown and menu.
- See "Design Specs" above.
- Does it match the Design?

# Technical Requirements 
- [ ] When in "Old" UI mode. These changes should not effect anything about how the UI looks or behaves. 
- [ ] Naming Convention. Consistent across all CSS Ids & classes - file and folders - Logic variable names...
- [ ] CSS: All new UI changes to CSS should include a class or id with a `coa-` prefix. This will ensure we do not change any other "under the hood" css that may use similar classes.

# Checklist:
